### PR TITLE
2 packages from project-everest/hacl-star at 0.3.2

### PR DESCRIPTION
**Pull request by victor-dumitrescu**
This pull-request concerns:
-`hacl-star.0.3.2`: OCaml API for EverCrypt/HACL*
-`hacl-star-raw.0.3.2`: Auto-generated low-level OCaml bindings for EverCrypt/HACL*

---
* Homepage: https://hacl-star.github.io/
* Source repo: git+https://github.com/project-everest/hacl-star.git
* Bug tracker: https://github.com/project-everest/hacl-star/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/hacl-star-raw/hacl-star-raw.0.3.2/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
+x-ci-accept-failures: ["centos-7" "oraclelinux-7"]
```
Comment (kit-ty-kate) [inline packages/hacl-star-raw/hacl-star-raw.0.3.2/opam]: I'm not sure where you go this file from but please keep the comment and `opam-version:` field first.

Comment (victor-dumitrescu) [reply packages/hacl-star-raw/hacl-star-raw.0.3.2/opam]: Ah, I'm sorry for the omission. My .opam file in the archive did preserve these comments so it's perhaps opam-publish which reformatted the file? Thanks for adding them back, I'll watch out for it next time.

Comment (kit-ty-kate) [reply packages/hacl-star-raw/hacl-star-raw.0.3.2/opam]: ok i reported it in https://github.com/ocaml-opam/opam-publish/issues/123 let's see if this gets fixed

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**